common/http/platform/beast/http.cpp

```diff
@@ -475,6 +478,9 @@ void Client::ConnectHandler(const error_code &ec, const asio::ip::tcp::endpoint
 		return;
 	}
 
+	boost::asio::socket_base::keep_alive option(true);
+	stream_->next_layer().set_option(option);
```

Comment:
Remember to use [the config option](https://github.com/mendersoftware/mender/blob/feature-c%2B%2B-client/common/config_parser.hpp#L51).